cmd/lakefs/cmd/migrate.go

```diff
@@ -52,6 +55,34 @@ func mustValidateSchemaVersion(ctx context.Context, kvStore kv.Store) int {
 	return version
 }
 
+func migrateImportACL(ctx context.Context, kvStore kv.Store) error {
+	policyKey := model.PolicyPath(acl.ACLPolicyName(acl.ACLWritersGroup))
+	p := model.PolicyData{}
+	_, err := kv.GetMsg(ctx, kvStore, model.PartitionKey, policyKey, &p)
+	if err != nil {
+		if errors.Is(err, kv.ErrNotFound) {
+			return nil // No policy - nothing to update
+		}
+	}
```

Comment:
- Can move the not found check outside the err check and return nil (as you did) - can also be done with nesting, I just prefer without if possible
- Return an error in case of an error to fail the migration of existing ACL group